tests/signing/tests.py

```diff
@@ -178,6 +178,39 @@ def test_invalid_sep(self):
             with self.assertRaisesMessage(ValueError, msg % sep):
                 signing.Signer(sep=sep)
 
+    def test_verify_with_non_default_key(self):
+        old_signer = signing.Signer('secret')
+        new_signer = signing.Signer('newsecret', fallback_keys=['secret'])
```

Comment:
I would add a test for multiple fallbacks:
```suggestion
        new_signer = signing.Signer('newsecret', fallback_keys=['othersecret', 'secret'])
```